src/App.js

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+import { 
+  Grommet,
+  Header, 
+  Main, 
+  Footer,
+  Anchor,
+  TextInput,
+  Box,
+  Paragraph,
+  Grid,
+  Card,
+  Button,
+  Form,
+  FormField,
+  Heading,
+  Image,
+  Text
+} from 'grommet';
+import {
+  Grommet as GrommetIcon,
+  Search,
+  ClearOption,
+  Configure,
+  Camera,
+  CloudUpload,
+  Selection,
+  BarChart,
+  FormNextLink,
+  Github,
+  Slack
+} from 'grommet-icons';
+import { grommetToolsTheme } from './theme.js';
+import GridCard from './components/GridCard.js';
+import SmallCard from './components/SmallCard.js';
+
+function App() {
+  return (
+    <Grommet className="App" theme={grommetToolsTheme}>
+
+      <Header height='72px'>
+        <Box 
+          justify='start' 
+          direction='row' 
+          gap='small'
+          >
+          <Box margin={{'left': 'medium'}}  align='center'>
+            <GrommetIcon size='36px' color="brand"/>
+          </Box>
+          <Anchor margin='xsmall' href="#" label="Grommet Tools"/>
+        </Box>
+        <Form>
+          <Box 
+            background='rgba(0, 0, 0, 0.04)' 
+            direction="row" 
+            justify='center' 
+            round
+            height='36px'
+            width='336px'
+            flex={{
+              grow: 336,
+              shrink: 100
+              }}
+            >
+            <FormField margin={{'bottom': '0px', 'left': '2px', 'top': '0px'}} border={false} name="search input">
```

Comment:
As far as I know, FormField doesn't have a `border` prop.

Reply:
Without setting the border to false there is a grey line under the formfield. On the documentation border isn't shown as a prop but it is shown in the theme section

Reply:
`border` could be set on the theme, it shouldn't be used as prop as there is no `border` prop to FormField.

Reply:
Changed border to be set on theme instead on on FormField